src/pages/link-app/index.tsx

```diff
@@ -0,0 +1,176 @@
+import {
+  // AspectRatio,
+  Button,
+  Container,
+  createStyles,
+  Flex,
+  Grid,
+  Stack,
+  Text,
+  Title,
+  Image,
+} from '@mantine/core';
+// import { YoutubeEmbed } from '~/components/YoutubeEmbed/YoutubeEmbed';
+import { containerQuery } from '~/utils/mantine-css-helpers';
+import { useCurrentUser } from '~/hooks/useCurrentUser';
+import { NextLink } from '@mantine/next';
+import Lottie from 'react-lottie';
+import * as linkAnimation from '~/utils/lotties/link-animation.json';
+
+export default function LinkApp() {
+  const { classes } = useStyles();
+  const currentUser = useCurrentUser();
+  const isMember = currentUser?.isMember;
+  const buttonData = {
+    text: isMember ? 'Download the Link App' : 'Become a member',
+    href: isMember ? 'https://github.com/civitai/civitai-link-desktop/releases/latest' : '/pricing',
+  };
+
+  return (
+    <Container size="md">
+      <Flex
+        direction={{ base: 'column', md: 'row' }}
+        justify="space-between"
+        className={classes.heroContainer}
+      >
+        <Stack spacing={12} mb={{ base: 24, md: 0 }}>
+          <Title className={classes.heroTitle}>Civitai Link App</Title>
+          {!isMember ? (
+            <Text className={classes.heroText}>❤️ Civitia Link is only available to members</Text>
+          ) : null}
+        </Stack>
+        <Flex align="center">
+          <Button
+            variant="filled"
+            color="blue"
+            size="lg"
+            radius="xl"
+            fullWidth
+            component={NextLink}
+            href={buttonData.href}
+          >
+            {buttonData.text}
+          </Button>
+        </Flex>
+      </Flex>
+
+      <Flex direction="row" className={classes.gradientContainer}>
+        <Flex justify="center" className={classes.videoBorder}>
+          <Lottie options={{ animationData: linkAnimation }} />
+        </Flex>
+        <div className={classes.gradientBox} />
+      </Flex>
+
+      <Stack spacing={12} mb={40}>
+        <Title className={classes.heading}>Add models to SD</Title>
```

Comment:
Might wanna add `order={3}` so it's treated as an `h3` here. Otherwise, it'll be considered h1 and compete with the title

Reply:
Oof I dont like that pattern. But will update it.